apps/dav/lib/BackgroundJob/PruneOutdatedSyncTokensJob.php

```diff
@@ -52,9 +52,10 @@ public function __construct(ITimeFactory $timeFactory, CalDavBackend $calDavBack
 
 	public function run($argument) {
 		$limit = max(1, (int) $this->config->getAppValue(Application::APP_ID, 'totalNumberOfSyncTokensToKeep', '10000'));
+		$retention = max(7, (int) $this->config->getAppValue(Application::APP_ID, 'syncTokensRetention', '60')) * 24 * 3600;
```

Comment:
```suggestion
		$retention = max(1, (int) $this->config->getAppValue(Application::APP_ID, 'syncTokensRetention', '60')) * 24 * 3600;
```
7 is arbitrary. If people change this value, let them do as they want :shrug:

Reply:
I don't mind much :)

Reply:
Mind my change or mind people setting up their instance for failure? :D

Reply:
putting a sane minimum